.github/workflows/label.yml

```diff
@@ -0,0 +1,17 @@
+# This workflow will triage pull requests and apply a label based on the
```

Comment:
Just asking -  isn't better for this file to be named labeler? and other file maybe label or something else?

Reply:
This is just the default name that github generates, could call it something different but will not be prioritising this PR until end of tomorrow